BrainPortal/app/views/custom_filters/_custom_filter_list.html.erb

```diff
@@ -18,20 +18,68 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.  
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 -%>
 
-<h4>By Custom Filter</h4>
 
-<%= overlay_ajax_link 'Create Custom Filter', {:controller => :custom_filters, :action => :new, :filter_class => @custom_filter.class.name} %>
+<li>
```

Comment:
The <ul>'s have attributes that are specific to the page, so they're not in the partial.